README.md

```diff
@@ -144,6 +144,9 @@ There are two types of configuration that `wrangler` uses: global user and per p
         - `*example.com/*`
         - `http://example.com/hello`
         This key is optional if you are using a workers.dev subdomain and is only required for `publish --release`.
+    - `kv-namespaces`: A listing of [Workers KV](https://workers.cloudflare.com/docs/reference/storage/overview/) namespace bindings that you want configured for your project, in uppercase format. _Note: Workers KV is available only to paid Workers users._ Examples:
+        - `[CACHE]`
+        - `[TODOS, USERS]`
```

Comment:
Don't you also need to specify the namespace ID?

Reply:
Yes correct, it's in #215. That said, it's currently in discussion I believe.